parastell/parastell.py

```diff
@@ -457,6 +412,63 @@ def export_dagmc(
                 **kwargs
             )
 
+    def build_full_model(self, invessel_build, magnet_coils, source_mesh):
+        """Use all of the input to make a complete stellarator model
+        with invessel components, magnets, and a source mesh
+        """
+
+        # In-Vessel Build
+        self.construct_invessel_build(
+            invessel_build['toroidal_angles'],
+            invessel_build['poloidal_angles'],
+            invessel_build['wall_s'],
+            invessel_build['radial_build'],
+            **invessel_build
+        )
+
+        # Magnet Coils
+        self.construct_magnets(
+            magnet_coils['coils_file'],
+            magnet_coils['cross_section'],
+            magnet_coils['toroidal_extent'],
+            **magnet_coils
+        )
+
+        # Source Mesh
+        self.construct_source_mesh(
+            source_mesh['mesh_size'],
+            source_mesh['toroidal_extent'],
+            **source_mesh
+        )
+
+    def export_full_model(self, export_dir, invessel_build, 
+                          magnet_coils, source_mesh, dagmc_export):
```

Comment:
We should probably include a docstring here. I'm happy to do so myself, if you prefer; I just need to figure out pushing commits to others' PRs...